libraries/CurieBLE/examples/LEDCentral/LEDCentral.ino

```diff
@@ -0,0 +1,158 @@
+/*
+  Copyright (c) 2016 Intel Corporation. All rights reserved.
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-
+  1301 USA
+*/
+
+// This example can work with CallbackLED to show the profile read/write operation in central
+#include <CurieBLE.h>
+
+struct bt_le_conn_param conn_param = {0x18, 0x28, 0, 400};
+
+const int ledPin = 13; // set ledPin to use on-board LED
+BLECentral bleCentral; // create central instance
+BLEPeripheralHelper *blePeripheral1 = NULL;
+
+BLEService ledService("19B10000-E8F2-537E-4F6C-D104768A1214"); // create service
+// create switch characteristic and allow remote device to read and write
+BLECharCharacteristic switchChar("19B10001-E8F2-537E-4F6C-D104768A1214", BLERead | BLEWrite);
+
+
+void bleCentralConnectHandler(BLEHelper& peripheral)
+{
+    // peripheral connected event handler
+    blePeripheral1 = (BLEPeripheralHelper *)(&peripheral);
+    Serial.print("Connected event, peripheral: ");
+    Serial.println(peripheral.address());
+    // Start discovery the profiles in peripheral device
+    blePeripheral1->discover();
+}
+
+void bleCentralDisconnectHandler(BLEHelper& peripheral)
+{
+    // peripheral disconnected event handler
+    blePeripheral1 = NULL;
+    Serial.print("Disconnected event, peripheral: ");
+    Serial.println(peripheral.address());
+    bleCentral.startScan();
+}
+
+void switchCharacteristicWritten(BLEHelper& peripheral, BLECharacteristic& characteristic)
+{
+    // Read response/Notification wrote new value to characteristic, update LED
+    Serial.print("Characteristic event, written: ");
+
+    if (switchChar.value()) 
+    {
+        Serial.println("LED on");
+        digitalWrite(ledPin, HIGH);
+    }
+    else
+    {
+        Serial.println("LED off");
+        digitalWrite(ledPin, LOW);
+    }
+}
+
+bool adv_found(uint8_t type, 
+               const uint8_t *data, 
+               uint8_t data_len,
+               void *user_data)
+{
+    bt_addr_le_t *addr = (bt_addr_le_t *)user_data;
+    int i;
+
+    pr_debug(LOG_MODULE_APP, "[AD]: %u data_len %u\n", type, data_len);
```

Comment:
Ok...  We need to distinguish the difference between pr_debug and Serial.print.  Somehow, need to make user to understand that pr_debug is for debugging purposes that log messages go out to another serial port.  Serial.print is the standard IO serial output for most users.  Any suggestions on how to make that clear?  May be a comment in the code?  That would be a whole lot of comments.  May be rename the routine a bit more obvious?